bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/model/local/EvChargerData.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solax.internal.model.local;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.binding.solax.internal.connectivity.rawdata.local.LocalConnectRawDataBean;
+import org.openhab.binding.solax.internal.model.InverterType;
+import org.openhab.binding.solax.internal.util.ByteUtil;
+
+/**
+ * The {@link EvChargerData} is an abstract class that contains the common information, applicable for all
+ * inverters.
```

Comment:
not an abstract class and also for chargers or generic devices, probably copy/paste without update :-)